buffer/src/main/java/io/netty/buffer/ByteBufUtil.java

```diff
@@ -58,6 +58,7 @@ protected CharBuffer initialValue() throws Exception {
 
     private static final int MAX_CHAR_BUFFER_SIZE;
     private static final int THREAD_LOCAL_BUFFER_SIZE;
+    public static final int MAX_UTF8_BYTES_PER_CHAR = 3;
```

Comment:
Just a nit... I wonder if we really want to expose this to the end user. Maybe just keep it private for now ?